### PR TITLE
fix: remove most of qt deprecation warnings

### PR DESCRIPTION
**Pull request by jedrzejowski**
I have made following changes:
 - `endl` -> `Qt::endl`
 - `left` -> `Qt::left`
 
This will remove almost every warning from compiler.

### CODE REVIEW

**Comment by brezerk** [comment]
hi @jedrzejowski 

Thanks a lot for PR. I'll review it today.

Thanks!

**Review by brezerk — CHANGES_REQUESTED** [review]
Looks good overall. Few items to address. Also not sure about `override` identifier.

File: src/q4wine-gui/prefixsettings.h [hunk 1]
```diff
@@ -127,7 +127,7 @@ class PrefixSettings : public QDialog, public Ui::PrefixSettings
         void cmdAddVersion_Click();
 
         void comboVersionList_Change(const QString & text);
-        void comboTemplatesList_Change(const int id);
+        void comboTemplatesList_Change(int id);
```
[the remark this answers is not in the document]
Comment (jedrzejowski) [reply src/q4wine-gui/prefixsettings.h:130]: I still do not understand why `const int`, it should be `int` or `const int&`.
When you are using `const int`, the value is being copied while executing.

[the remark this answers is not in the document]
Comment (brezerk) [reply src/q4wine-gui/prefixsettings.h:130]: you are quite right, however you should update both constructor and function to match in this case. unless I am missing something :)

**Review by brezerk — APPROVED** [review]

**Comment by brezerk** [comment]
Looks good. Thanks a lot! 👍

**Comment by tehnick** [comment]
You broke support of Ubuntu Focal and older:
https://launchpad.net/~tehnick/+archive/ubuntu/q4wine-devel/+packages
As well as all systems wit Qt < 5.15.
Is it worth it?

**Comment by brezerk** [comment]
> error: ‘endl’ is not a member of ‘Qt’

Yeah, not that good. Thank you for a note.

пн, 10 янв. 2022 г., 01:31 Boris Pek ***@***.***>:

**Comment by tehnick** [comment]
Just reminding that build for Ubuntu Focal still not fixed yet. Example of build log is [here](https://launchpadlibrarian.net/579385847/buildlog_ubuntu-focal-amd64.q4wine-devel_1.3.13+2+g9735939-0ppa1~focal1_BUILDING.txt.gz). Example of error:
```
/<<BUILDDIR>>/q4wine-devel-1.3.13+2+g9735939/src/q4wine-lib/q4wine-lib.cpp: In member function ‘void corelib::getBuildFlags()’:
/<<BUILDDIR>>/q4wine-devel-1.3.13+2+g9735939/src/q4wine-lib/q4wine-lib.cpp:574:53: error: ‘endl’ is not a member of ‘Qt’
  574 |     Qcout<<QObject::tr("Buildtime flags are:")<<Qt::endl<<Qt::endl;
```

Ubuntu Focal (20.04 LTS) uses Qt 5.12.8.

**Comment by tehnick** [comment]
I have re-checked: even Ubuntu Bionic (18.04 LTS) was supported until this PR.
https://launchpad.net/~tehnick/+archive/ubuntu/q4wine/+packages
https://launchpad.net/~tehnick/+archive/ubuntu/q4wine-devel/+packages

**Pull request merged**